wiki/Mapping_Techniques/Overlap/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overlap
+
+![overlap.png](./img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```

Comment:
The alt text should be a description of the image. If you don't have one, it should be left blank.

Reply:
what more would you add to this? the curvature of the slider? it simply is a hitcircle overlapping a sliderhead

Reply:
Just to be sure we are talking about the same thing,
```
![overlap.png](./img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
  ^ This is the alt text          ^ This is the title/hover text, also used as the caption on web
```

Alt text only displays when the image doesn't load, but it is also used for accessibility purposes too. If you don't have an image description for the alt text, then don't put anything in there.

Reply:
that clears it up. so leave it blank?

Reply:
Yeah, that'll be fine.